pkgs/top-level/python-packages.nix

```
@@ -18178,7 +18178,7 @@ self: super: with self; {
 
   vtjp = callPackage ../development/python-modules/vtjp { };
 
-  vtk = toPythonModule (pkgs.vtk_9.override {
+  vtk = toPythonModule (pkgs.vtk_9_2.override {
```

```suggestion
  vtk = toPythonModule (pkgs.vtk_9.override {
```
should work once https://github.com/NixOS/nixpkgs/pull/401727 is merged.